tests/components/steamist/test_sensor.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for the steamist sensos."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant
+
+from . import (
+    MOCK_ASYNC_GET_STATUS_ACTIVE,
+    MOCK_ASYNC_GET_STATUS_INACTIVE,
+    _async_setup_entry_with_status,
+)
+
+
+async def test_steam_active(hass: HomeAssistant) -> None:
```

Comment:
maybe just verify native unit of measurement here

Reply:
4b3891a1bc3bef7bb9134bb3fe64be5cb4103a1c